src/main/docs/guide/azureCosmos/azureCosmosQuickStart.adoc

```diff
@@ -0,0 +1,67 @@
+At this point there is still no ability to create Micronaut project with Azure Cosmos support via Micronaut Launch. Our team will be working on it in the near future.
+
+To get started with Micronaut Data Azure Cosmos add the following dependency to your annotation processor path:
+
+dependency:io.micronaut.data:micronaut-data-document-processor[scope="annotationProcessor""]
+
+NOTE: For Kotlin the dependency should be in the `kapt` scope and for Groovy it should be in `compileOnly` scope.
+
+You should then configure a compile-scoped dependency on the `micronaut-data-azure-cosmos` module:
+
+dependency:io.micronaut.data:micronaut-data-azure-cosmos[]
+
+Next up you need to configure at least one data source. The following snippet from `application.yml` is an example of configuring the default Azure Cosmos Db data source:
+
+Example YAML configuration
+[source,yaml]
+----
+include::doc-examples/azure-cosmos-example-java/src/main/resources/application.yml[]
+----
+
+To retrieve objects from the database you need to define a class annotated with ann:data.annotation.MappedEntity[]:
+
+snippet::example.Book[project-base="doc-examples/azure-cosmos-example", source="main", tags="book"]
+
+Followed by an interface that extends from api:data.repository.CrudRepository[]
+
+snippet::example.BookRepository[project-base="doc-examples/azure-cosmos-example", source="main", tags="repository"]
+
+<1> The interface is annotated with ann:io.micronaut.data.cosmos.annotation.CosmosRepository[]
+<2> The `CrudRepository` interface take 2 generic arguments, the entity type (in this case `Book`) and the ID type (in this case `String`)
+
+You can now perform CRUD (Create, Read, Update, Delete) operations on the entity. The implementation of `example.BookRepository` is created at compilation time. To obtain a reference to it simply inject the bean:
+
+snippet::example.BookRepositorySpec[project-base="doc-examples/azure-cosmos-example", tags="inject", indent="0"]
+
+In Azure Cosmos world, every `MappedEntity` will correspond to the container. One container can hold only one entity or document type.
+`MappedEntity` name will be used to obtain container name. If entity class is `CosmosBook` then expected container name will be `cosmos_book` unless not overriden in `MappedEntity` annotation value. Default naming strategy for entity fields is `Raw` strategy and users should not usually need to override it.
```

Comment:
Rephrase `MappedEntity name will be used to obtain container name` to `The simple name of the class annotated with @MappedEntity will be used as the container name by default.`